plugins/webhooks/index.js

```diff
@@ -617,7 +645,8 @@ exports.register = (server, options, next) => {
     const pluginOptions = joi.attempt(options, joi.object().keys({
         username: joi.string().required(),
         ignoreCommitsBy: joi.array().items(joi.string()).optional(),
-        restrictPR: joi.string().valid('all', 'none', 'branch', 'fork').optional()
+        restrictPR: joi.string().valid('all', 'none', 'branch', 'fork').optional(),
+        chainPR: joi.boolean().optional()
```

Comment:
should add param `chainPR` to doc comment.

Reply:
I added it. Thanks.